sites/fast-website/src/public/index.html

```diff
@@ -89,7 +89,57 @@
         <!-- documentation:start -->
         <section class="section">
           <div class="section-component">
-            Documentation components
+            <site-card-section>
+              <site-feature-card>
+                <h5 slot="header">01</h5>
```

Comment:
I wouldn't expect an <h5> to precede an <h4> for semantic and accessibility?

Reply:
Div has no semantics, so you aren't going to get your intended structure. @awentzel is the recommendation here to have this be an h4 as well? We technically can take this as a follow-up, but we should absolutely ensure that the HTML markup is both semantic and valid, which I believe is @awentzel's point here.